llvm/lib/Transforms/Utils/ProfileVerify.cpp

```diff
@@ -26,6 +26,12 @@ using namespace llvm;
 static cl::opt<int64_t>
     DefaultFunctionEntryCount("profcheck-default-function-entry-count",
                               cl::init(1000));
+static cl::opt<bool> AnnotateSelect("profcheck-annotate-select",
```

Comment:
Why do we need an option to control whether we check the select annotation or not?

Also all these options are missing descriptions. Can you add a sentence for each?